src/content/docs/fr/guides/imports.mdx

```diff
@@ -1,38 +1,46 @@
 ---
-title: Fichiers Statiques
-description: Apprenez à importer différents types de contenu avec Astro.
+title: Imports
+description: Apprendre comment importer différents types de contenu avec Astro.
+i18nReady: true
 ---
+import RecipeLinks from "~/components/RecipeLinks.astro";
 
 Astro supporte la plupart des ressources statiques sans aucune configuration requise. Vous pouvez utiliser la directive `import` n'importe où dans votre projet JavaScript (y compris votre *frontmatter* d'Astro) et Astro inclura une copie construite et optimisée de cette ressource statique dans votre build final. `@import` est également prise en charge dans les balises CSS et `<style>`.
 
+Astro prend en charge la plupart des ressources statiques sans aucune configuration requise. Vous pouvez utiliser l'instruction `import` n'importe où dans le JavaScript de votre projet (y compris dans le frontmatter d'Astro) et Astro inclura une copie construite et optimisée de cette ressource statique dans votre version finale. `@import` est également supporté à l'intérieur des balises CSS & `<style>`.
+
 ## Types de fichiers supportés
 
-Les types de fichiers suivants sont pris en charge d'emblée par Astro :
+Les types de fichiers suivants sont pris en charge par Astro :
 
 - Composants Astro (`.astro`)
-- Markdown (`.md`)
+- Markdown (`.md`, `.markdown`, etc.)
 - JavaScript (`.js`, `.mjs`)
 - TypeScript (`.ts`, `.tsx`)
-- Packages NPM
+- Paquets NPM
 - JSON (`.json`)
 - JSX (`.jsx`, `.tsx`)
 - CSS (`.css`)
 - Modules CSS (`.module.css`)
-- Images & éléments (`.svg`, `.jpg`, `.png`, etc.)
+- Images et actifs (`.svg`, `.jpg`, `.png`, etc.)
+
+De plus, vous pouvez étendre Astro pour ajouter le support de différents [Frameworks UI ](/fr/core-concepts/framework-components/) comme React, Svelte et Vue. Vous pouvez également installer l'[intégration Astro MDX](/fr/guides/integrations-guide/mdx/) et utiliser les fichiers `.mdx` dans votre projet.
+
+### Les fichiers dans `public/`
 
-Si vous ne trouvez pas le type de ressource que vous recherchez, consultez notre [Bibliothèque d'intégrations](https://astro.build/integrations/). Vous pouvez étendre Astro pour ajouter la prise en charge de différents types de fichiers, comme les composants Svelte et Vue.
+Vous pouvez placer n'importe quelle ressource statique dans le répertoire [`public/`](/fr/core-concepts/project-structure/#public) de votre projet, et Astro le copiera directement dans votre version finale sans y toucher. Les fichiers dans le répertoire `public/` ne sont pas construits ou regroupés par Astro, ce qui signifie que n'importe quel type de fichier est supporté. Vous pouvez référencer un fichier `public/` par un chemin URL directement dans vos modèles HTML.
```

Comment:
`par un chemin URL` sounds weird. I don't really have other suggestion, so don't take mine but I don't know if you have any idea.

```suggestion
Vous pouvez placer n'importe quelle ressource statique dans le répertoire [`public/`](/fr/core-concepts/project-structure/#public) de votre projet, et Astro le copiera directement dans votre version finale sans y toucher. Les fichiers dans le répertoire `public/` ne sont pas construits ou regroupés par Astro, ce qui signifie que n'importe quel type de fichier est supporté. Vous pouvez référencer un fichier `public/` par un chemin directement dans vos modèles HTML.
```

or maybe:
```suggestion
Vous pouvez placer n'importe quelle ressource statique dans le répertoire [`public/`](/fr/core-concepts/project-structure/#public) de votre projet, et Astro le copiera directement dans votre version finale sans y toucher. Les fichiers dans le répertoire `public/` ne sont pas construits ou regroupés par Astro, ce qui signifie que n'importe quel type de fichier est supporté. Vous pouvez référencer un fichier `public/` par un chemin d'URL directement dans vos modèles HTML.
```